### PR TITLE
TemplateEditor backpressed() and other functions improved.

### PR DESCRIPTION
**Pull request by free4murad**
@opticod 
I have solved issue #245.
Please have a look.

### CODE REVIEW

**Comment by opticod** [comment]
It looks like, you haven't updated your branch with upstream while creating PR.

**Comment by opticod** [comment]
It will automatically resolve the merge conflict

**Comment by free4murad** [comment]
I will solve the merge conflict

File: source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java [hunk 1]
```diff
@@ -387,29 +391,42 @@ public void onClick(View v) {
                 });
 
                 break;
+
```
Comment (opticod) [inline source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java]: Don't add unnecessary void lines.

**Comment by free4murad** [comment]
@opticod 
will make a new PR.

**Pull request closed without merging**